src/FileManager.php

```diff
@@ -174,6 +176,11 @@ public function getRootDirectory(): string
         return $this->rootDirectory;
     }
 
+    public function getTemplatesFolder(): string
```

Comment:
In Symfony we always use `Dir` instead of `Folder` or `Directory`, so please rename this method to `getTemplatesDir()`. Thanks!